Get set up locally and write up instructions

- [x] Get local version running
- [x] Update README/Wiki

```
ben@ben-work-laptop:LocalSpend Web App$ npm start

> localloop-web@0.1.13 start /home/ben/Documents/Personal/Volunteering/LocalSpend/Code/LocalSpend Web App
> ng serve

An unhandled exception occurred: Job name "..getProjectMetadata" does not exist.
See "/tmp/ng-BM6hI4/angular-errors.log" for further details.
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! localloop-web@0.1.13 start: `ng serve`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the localloop-web@0.1.13 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ben/.npm/_logs/2020-07-29T15_10_17_548Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -6,8 +6,8 @@ This repository contains the Web application for the LocalSpend system.
 
 | Branch | Status |
 | --- | --- |
-| Master: | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=master)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
-| Development: | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=development)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
+| `master` | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=master)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
+| `development` | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=development)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
 
 ## Contents
 

--- a/README.md
+++ b/README.md
@@ -37,6 +37,7 @@ To get this repository set up:
   - `npm install`
 3. Start the application:
   - `npm start`
+  - The app. will automatically reload after source file changes
 
 [LocalLoop-Server]:https://github.com/Pear-Trading/Foodloop-Server
 [tj/n]:https://github.com/tj/n
@@ -46,6 +47,10 @@ To get this repository set up:
 
 The app defaults to using the development server. For other options, see 
 `src/environments/environments.ts`.
+
+## Build
+
+Run `ng build` to build the project; the resulting files will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
 
 ## Testing
 
@@ -89,37 +94,3 @@ The interface itself is based off of [CoreUI][core-ui] which is MIT Licenced.
 For information, see `LICENCE.MIT` included in this repo.
 
 [core-ui]: http://coreui.io
-
-___
-
-## Old README
-
-This is the old README with some minor tips on getting started, and more reading.
-
-### Angular2DevelopmentCLI
-
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.0.0-beta.32.3.
-
-### Development server
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
-
-### Code scaffolding
-
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive/pipe/service/class/module`.
-
-### Build
-
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
-
-### Running unit tests
-
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
-
-### Running end-to-end tests
-
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
-Before running the tests make sure you are serving the app via `ng serve`.
-
-### Further help
-
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).

--- a/README.md
+++ b/README.md
@@ -1,30 +1,34 @@
-# LocalLoop Web Interface
+# LocalSpend - Web Application
 
-This is the repository for the LocalLoop web interface, for traders and
-customers to see and submit data to the service.
+This repository contains the Web application for the LocalSpend system.
+
+## Current Status
 
 | Branch | Status |
 | --- | --- |
-| *Master:* | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=master)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
-| *Development:* | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=development)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
+| Master: | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=master)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
+| Development: | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=development)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
 
 ## Getting Started
 
-To get started with development, you will need an up to date version of
-node.js, git, and access to either the backend dev server, or a local running
-copy of the backend server from [LocalLoop Server][LocalLoop-Server].
+To get started with development, you will need an up-to-date version of
+Node.js, git, and access to either the backend dev. server or a local
+copy of it from the [LocalSpend Server][LocalLoop-Server] repo.
 
-For your local node.js, We reccomend using [n][tj/n] on \*nix and Mac, for
-Windows take a look at [nodist][marcelklehr/nodist] - although other options
-exist. We reccomend Node.js version 8.0.0+ and npm version 5.3.0+.
+For your local Node.js, we recommend using:
+- [n][tj/n] for \*nix and Mac; and
+- [nodist][marcelklehr/nodist] for Windows.
 
-To get this repository set up, first clone it and then run the following
-commands:
+We reccomend Node.js version 8.0.0+ and npm version 5.3.0+.
 
-```
-npm install -g @angular/cli
-npm install
-```
+To get this repository set up:
+
+1. Clone it
+1. Install the dependencies:
+  - `npm install -g @angular/cli`
+  - `npm install`
+1. Start the application:
+  - `npm start`
 
 [LocalLoop-Server]:https://github.com/Pear-Trading/Foodloop-Server
 [tj/n]:https://github.com/tj/n
@@ -32,12 +36,14 @@ npm install
 
 ## Environments
 
-The app defaults to using the development server. For other options, see
-`src/environments/environments.ts`
+The app defaults to using the development server. For other options, see 
+`src/environments/environments.ts`.
 
 ## Testing
 
-There are two types of test in this codebase, unit tests (using Karma) and E2E tests (using Protractor).
+There are two types of test in this codebase:
+- unit tests (using Karma); and
+- E2E tests (using Protractor).
 
 ### Karma Unit Tests
 
@@ -47,15 +53,25 @@ These tests are a long-running process, and will automatically run on file chang
 
 ### Protractor E2E Tests
 
-To run these, you will need to run the following:
+To run these, you will need to run the following command:
 
-```
-webdriver-manager update
-```
+- `webdriver-manager update`
 
-This will download the correct webdriver for you (we use chromedriver), and any other dependencies it needs. After that, you can run the tests with `npm run e2e`. Note these run once, and will need to be re-run every time you want to run an e2e test. You will also need to run the 
+This will download the correct webdriver for you (we use chromedriver), and 
+any other dependencies it needs.
+After that, you can run the tests with `npm run e2e`.
+Note these run once, and will need to be re-run every time you want to run an 
+e2e test.
 
-These tests are best run regularly, and should show if any issues have emerged in other parts of the application that you are not aware of, or if some part of the flow through the app has changed sufficiently. 
+These tests are best run regularly, and should show if any issues have emerged 
+in other parts of the application that you are not aware of, or if some part 
+of the flow through the app has changed significantly. 
+
+## Troubleshooting
+
+### ‘Error: EACCES: permission denied, access '/usr/local/lib'’ when installing dependencies
+
+Change npm's default directory by following [these steps](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory).
 
 ## Licences
 
@@ -66,11 +82,13 @@ For information, see `LICENCE.MIT` included in this repo.
 
 [core-ui]: http://coreui.io
 
-# Old README
+___
+
+## Old README
 
 This is the old README with some minor tips on getting started, and more reading.
 
-## Angular2DevelopmentCLI
+### Angular2DevelopmentCLI
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.0.0-beta.32.3.
 

--- a/README.md
+++ b/README.md
@@ -9,6 +9,14 @@ This repository contains the Web application for the LocalSpend system.
 | Master: | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=master)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
 | Development: | [![Build Status](https://travis-ci.org/Pear-Trading/FoodLoop-Web.svg?branch=development)](https://travis-ci.org/Pear-Trading/FoodLoop-Web) |
 
+## Contents
+
+1. [Getting Started](#getting-started)
+1. [Environments](#environments)
+1. [Testing](#testing)
+1. [Troubleshooting](#troubleshooting)
+1. [Licences](#licenses)
+
 ## Getting Started
 
 To get started with development, you will need an up-to-date version of
@@ -24,10 +32,10 @@ We reccomend Node.js version 8.0.0+ and npm version 5.3.0+.
 To get this repository set up:
 
 1. Clone it
-1. Install the dependencies:
+2. Install the dependencies:
   - `npm install -g @angular/cli`
   - `npm install`
-1. Start the application:
+3. Start the application:
   - `npm start`
 
 [LocalLoop-Server]:https://github.com/Pear-Trading/Foodloop-Server

